Error in Typescript Types

Importing charts in modern Typescript (5.x +, node 16, using default bundler) results in a types not found error. Specifically:

error TS7016: Could not find a declaration file for module 'vue3-apexcharts'. '.../dist/vue3-apexcharts.js' implicitly has an 'any' type.
  There are types at '.../vue3-apexcharts/typings/vue-apexcharts.d.ts', but this result could not be resolved when respecting package.json "exports". The 'vue3-apexcharts' library may need to update its package.json or typings.

You can use https://arethetypeswrong.github.io/?p=vue3-apexcharts to verify errors & see how TS if viewing the lib. See also https://github.com/microsoft/TypeScript/issues/52363 for a discussion of the issue within the TypeScript team.

Desired fix: please set the correct explicit typings, at least for when using JS modules (since that's what I use...).

--- a/package.json
+++ b/package.json
@@ -7,6 +7,7 @@
   "private": false,
   "exports": {
     ".": {
+      "types": "./typings/vue-apexcharts.d.ts",
       "import": "./dist/vue3-apexcharts.js",
       "require": "./dist/vue3-apexcharts.umd.cjs"
     }

